### DISCUSSION TITLE
CVE-2020-15999 Virus found in harfbuzz-2.7.4.tar.gz

### DISCUSSION THREAD

**Discussion by jhonlinemail** [Homebrew/discussions#883] [Everyday usage]
Hello Brew Community,

I am new here (first post), and hoping you might direct me to the proper method to alert the developers of a possible virus I found this morning?

Details:

File:
511c258c38a3e026cdfd5bd1d3e94f346d053b82b7acc487905cd564b8d43e5c--harfbuzz-2.7.4.tar.gz

Location: 
/Library/caches/Homebrew/downloads

Virustotal.com Results:
12/58 virus detected
Exploit.CVE-2020-15999

Date:
2021-02-21

Info on this exploit:
https://attivonetworks.com/privilege-escalation-cve-2020-17087-cve-2020-15999/

Thank you!

![Screenshot-Virusscan-CVE-2020-15999-harfbuzz-2 7 4 tar gz-2021-02-22](https://user-images.githubusercontent.com/79477652/108759951-46a0d400-750a-11eb-9fba-c152abb16445.png)

**Comment by Bo98** [comment]
Nothing to be alarmed of: https://github.com/harfbuzz/harfbuzz/issues/2750

It's finding a file that used in tests to make sure a certain font exploit (CVE-2020-15999) is fixed. However, some anti-virus software think that it's there to be used to attack vulnerable systems where the exploit is unfixed - but it's not.

It's a false positive, but the next version of HarfBuzz is going to remove the file anyway. This is also only in the downloaded source achive (presumably you installed HarfBuzz from source) - the file isn't actually installed in the end, nor does it exist in any of our bottles.
